### DISCUSSION TITLE
M4B Chapters

### DISCUSSION THREAD

**Discussion by trymeouteh** [puddletag/puddletag#757] [Ideas]
Please add the ability to add, edit and remove chapters in M4B files. M4B files are audiobook files.

**Comment by sandrotosi** [comment]
i have no experience with the M4B format, can you elaborate more how this could be a useful addition to puddletag, which we describe as a "Powerful, simple, audio tag editor for GNU/Linux", and not an audio editor, which appears what youre after

**Reply by trymeouteh** [reply]
The chapters are markers for when in an audio file a chapter is. For example the preface chapter is likely going to be 0:00:00, chapter 1 could be something like 0:24:37, etc. This is not editing the audio itself but is metadata on the audio file.

There is not a single FOSS GUI M4B tag editor that can edit chapters in a M4B file. Puddletag could be the first and give FOSS software users a tool with a GUI to edit chapters and tags of a M4B audiobook file.
